service-registry/src/main/java/io/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -103,7 +109,7 @@ private void checkSchemaIdSet() {
                     microservice.getVersion(),
                     localSchemas,
                     existSchemas);
-            return;
+            return true;
```

Comment:
this flag  just  judges whether get the microservice schema dates successfully,  another flag "schemaIdSetMatch" judges whether the schema match.